lib/command.js

```diff
@@ -0,0 +1,20 @@
+(function (mson2json, fs) {
+
+    'use strict';
+
+    function loadMson(path, cb) {
+        fs.readFile(path, function (err, data) {
+            if (err) throw err;
+            return cb(data.toString());
+        });
+    }
+
+    module.exports = function run(path, cb) {
+        return loadMson(path, function (msonData) {
+            mson2json(msonData, cb)
+        })
+    };
+})(
+    require('./mson2json'),
+    require('fs')
+)
```

Comment:
should we maybe just combine this logic with the bin mson2json file? that way lib/mson2json.js only takes a buffer, whereas bin/mson2json only takes a file path.